src/content/autoscan.cc

```diff
@@ -34,6 +34,8 @@
 #include "content_manager.h"
 #include "database/database.h"
 
+#include <fmt/ostream.h>
```

Comment:
Why is it necessary. It must be before the other includes.

Reply:
Sorry, that was a bullshit commit. This should be a proper fix.